tools/spm/generate_partition_code.py

```diff
@@ -38,7 +38,7 @@
                     if '_NAME_' not in filename]
 MANIFEST_FILE_PATTERN = '*_psa.json'
 MBED_OS_ROOT = os.path.abspath(path_join(SCRIPT_DIR, os.pardir, os.pardir))
-SPM_CORE_ROOT = path_join(MBED_OS_ROOT, 'components', 'TARGET_PSA', 'spm')
+SPM_CORE_ROOT = path_join(MBED_OS_ROOT, 'components', 'TARGET_PSA', 'TARGET_MBED_SPM')
```

Comment:
Any concerns with this @theotherjimmy? Thinking about the online compiler. However it may just be limited to the FUTURE_SEQUANA target.

Reply:
@orenc17 Also, making sure this doesn't break things inadvertently.
@lrusinowicz thoughts?

(Should be fine, just being cautious)